source/sdk/Text.ooc

```diff
@@ -190,6 +190,17 @@ Text: cover {
 		this free(Owner Receiver)
 		result
 	}
+	nullTerminated: func -> This {
```

Comment:
So how do you plan to pass a content of Text to the system calls which expects `\0` at the end of char buffer ?